base/iostream.jl

```diff
@@ -144,7 +144,10 @@ end
 Apply the function `f` to the result of `open(args...)` and close the resulting file
 descriptor upon completion.
 
-**Example**: `open(readstring, "file.txt")`
+# Examples
+```julia-repl
```

Comment:
`julia-repl` blocks are for things that look like REPL sessions, i.e. have `julia>` input lines and lines of output. Since this doesn't have a "prompt" in front of it, I think it should just be `julia` instead of `julia-repl`.